### ISSUE TITLE
Doesn't work unfortunately.

### ISSUE THREAD

**Issue by khanpearl** [c-rainbow/twitch-audio-web#26]
When the radio icon is clicked, it does not work anymore. Just complete silence.

EDIT: It started working, wasn't working for a while now. Thank you for the extension so much!

**Comment by c-rainbow** [comment]
Hello @khanpearl ,

Thanks for the report. I already submitted the fix (v0.6.0) for review and am waiting for the result.

To give you some background, Twitch sends a silent video (or audio) during the advertisement. So, if you are lucky and don't need to wait for the ads, the audio will be played immediately. With ads, the audio will be still played normally in 15-30 seconds.

I think the way I used to bypass the ads in previous versions is not working anymore.

The new version (v0.6.0) unfortunately plays the ads in the radio mode, which I think is better than just 30 seconds of silence, but instead it should now skip the ads if you subscribe the channel or have Twitch Prime.
